definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent-resp-sol-can-complete-aos-nonprod.json

```diff
@@ -0,0 +1,9 @@
+[
```

Comment:
please use `-resp-journey-nonprod`

1 flag per 1 release.

We will have only 3 releases:
- today
- RRJ `-resp-journey-nonprod`
- SaC `-share-a-case-nonprod`

FR is going to have bailiff as well.